app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -216,8 +214,8 @@ private fun HomeScreenContent(
                 refreshing = isRefreshing,
                 state = refreshingState,
                 modifier = Modifier.align(alignment = Alignment.TopCenter),
-                backgroundColor = MaterialTheme.colors.pullRefreshBackgroundColor,
-                contentColor = Color.CaribbeanGreen
+                backgroundColor = AppTheme.colors.pullRefreshBackgroundColor,
+                contentColor = CaribbeanGreen
```

Comment:
similar idea as the above comment, it will be the best practice to unify all color reference from `AppTheme.colors`. Imagine 1 day you want to change a color that appears in multiple places, it will be easier to manage at the theme level rather than going to each section to change this 🤓